tests/unit/cluster/info.tcl

```diff
@@ -101,3 +101,44 @@ start_cluster 3 0 {tags {external:skip cluster} overrides {cluster-node-timeout
         }
     }
 }
+
+start_cluster 2 0 {tags {cluster}} {
```

Comment:
Let's put the test in the start_cluster we created above. It is a simple test and we can save a start_cluster block.

I am also suggesting that we only wirte a simple test, that is, we execute it in a multi-exec, and then check if cluster_stats_messages_xxx is 0. BTW, let's also reset total_cluster_links_buffer_limit_exceeded too